contrib/pipelines/core/store_s3.go

```diff
@@ -247,6 +247,16 @@ func NewStoreS3FromConfig(cfg *viper.Viper) (*StoreS3, error) {
 	maxFlowsPerObject := cfg.GetInt(CfgRoot + "store.s3.max_flows_per_object")
 	maxSecondsPerObject := cfg.GetInt(CfgRoot + "store.s3.max_seconds_per_object")
 	maxSecondsPerStream := cfg.GetInt(CfgRoot + "store.s3.max_seconds_per_stream")
+	if maxFlowArraySize <= 0 {
```

Comment:
you can change to `cfg.GetUint()` and then there is no need for checking for an incorrect value. https://godoc.org/github.com/spf13/viper#GetUint